db/migrate/20180701234758_add_role_to_users.rb

```diff
@@ -0,0 +1,24 @@
+class AddRoleToUsers < ActiveRecord::Migration
+
+  # 0 = nobody, 100 = admin.  See enum in models/user.rb
+  # There must be a better way to do this
+
+  @@user = 10
```

Comment:
You can actually get this right off of the class. i.e.: `User.roles[:user]`. ...You could also avoid your custom integers that way!

Reply:
OK. (The scoping & visibility rules remain a mystery for me.)